app/views/users/show.html.erb

```diff
@@ -16,22 +17,30 @@
       <% if @posts.empty? %>
         <p>There are no posts for this user</p>
       <% else %>
-        <h2>Take a look at all of the users and their number of posts</h2>
-
         <% @posts.each do |post| %>
           <div class="post-card">
             <h3>Post #<%= post.id %>: <%= post.title %></h2>
             <p class="post-card-d"><%= post.text %></p>
             <p class="post-card-a">
-              Comments: <%= post.comment_counter %>, Likes: <%= post.like_counter %>
+              Comments: <%= post.comments_counter %>, Likes: <%= post.likes_counter %>
             </p>
           </div>
```

Comment:
[OPTIONAL] The user should be able to navigate to the post-show page by clicking on the post title. I recommend implementing that for a better user experience.